packages/datadog_flutter_plugin/test/tracing/tracing_headers_test.dart

```diff
@@ -6,10 +6,35 @@ import 'package:datadog_flutter_plugin/src/tracing/tracing_headers.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
-  test('generateTracingContext generates 64-bit values', () {
+  test('TracingIdRepresentation generates proper values', () {
+    // Create a value in 128-bit hex that has leading zeros on both
+    // the low and high 64-bits, and ensure we get the proper values
+    const low = 0x01445feed89934bb;
+    const high = 0x01222f00d89934ba;
+
+    var combined = BigInt.from(high);
+    combined = (combined << 64) + BigInt.from(low);
+
+    final tracingId = TracingId(combined);
+
+    expect(tracingId.asString(TracingIdRepresentation.hex),
+        '1222f00d89934ba01445feed89934bb');
```

Comment:
/blocking

the generated id looks off here to me. 

Taking from one of my examples
`6634a5530000000035ad5c814c5a1e78`

first 32 bits: timestamp: 6634a553 (hex) -> 1714726227 (dec) ->  Friday, 3 May 2024 08:50:27
second 32 bits: 00000000
last 64 bits: 35ad5c814c5a1e78 (pure random)

we should add some test case on `_generateTraceId` method.

Reply:
For reference https://github.com/DataDog/dd-sdk-ios/blob/develop/DatadogInternal/Tests/NetworkInstrumentation/TraceIDGeneratorTests.swift#L17

Reply:
This essentially only testing just the various string representations that can be created from a `BigInt`, and that low / high bits get masked out and padded as appropriate. The low / high bits that are being passed in aren't conformant to the trace id spec.

That said, you're right should add a test that checks that `TracingId` creates a conformant Id when asked.